pyproject.toml

```diff
@@ -27,6 +27,7 @@ classifiers = [
 dependencies = [
     "certifi==2024.7.4",
     "grpcio==1.62.1",
+    "Brotli==1.1.0",
```

Comment:
Could we add this also to https://github.com/netboxlabs/diode-netbox-plugin/blob/develop/docker/requirements-diode-netbox-plugin.txt, please? When running local version of netbox with mounted diode plugin we install requirements from this separate file (not sure it's possible to extract from pyproject.toml directly in the Dockerfile we use.